Program to convert RawROOT files into LCIO format

HecGC opened the pull request:
It includes the converter in eventbuilding and a quick processor to display the total NHits in Analysis. 
It may be possible that there are many bugs with the algorithms but so far it runs. If there are any issues compiling let me know.

HecGC committed:
First commit

HecGC committed:
Remove extra files

airqui commented:
Thanks!
Could you maybe remove the mapping, masked, pedestal etc files from the PR ?
I think that it make sense to keep them in a separated repository (or even not there but in a private copy from eos, for example).

airqui commented:
Hi @HecGC 
we have made some changes in the pedestal and mip calibration files... with more entries for the pedestals.
The latest files are in the eos. And we should use the files with the method2 tag in the name.
Could you check if this is still compatible with your analysis ? The mip files are probably okay, but not the pedestal files,

For the pedestal file, I provide average of the pedestal for the sca without data or with failed fits. In this case, I tag them by writing ped_error_<5

For the mips, if there is not enough hits to calculate the mip, I write all zero's.  These are channels uncalibrated (and most probably masked).

In the case that we have enough hits but somehow the fit failed, I write a -5 in the second value. In this way we have them identified and, if needed, we use for them an averaged mip value (averaged over all channels in the same chip)

HecGC committed:
Latests updates:
	Adapt the code to accept the method2 pedestals and calibration files
	improvementes in the algorithm
	the event builder now creates a logroot
	removal of commissioning folders

HecGC commented:
I have taken into account the new pedestal and calibration files. Also there is a line in Layer 8, chip 3, channel 7 that has a lot of -10 as error in the pedestals. I do not know if that is intended.
Then I removed the commissioning files excep the config folder because I created it to map the layers, tungsten values and if there is a cob among them.
